api/src/Streams.cpp

```diff
@@ -58,7 +58,7 @@ void Stream::WriteJSON(
   writer->Key("live");
   writer->Bool(live_);
   writer->Key("nsfw");
-  writer->Bool(nsfw_);
+  writer->Bool(service_nsfw_);
```

Comment:
@slugalisk is this what you mean on streams? or another value

Reply:
it should be `nsfw_ || service_nsfw_`

Reply:
lines 15 and 61